build.tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  /* extend main config to add configs needed only at build time */
```

Comment:
This makes the file an invalid json; are we sure we want to keep it?

Reply:
I believe it clarifies the purposes of defining specific properties. Nonetheless, it should be sufficiently understandable even without them.